### PR TITLE
{Release} Upgrade to Azure CLI 2.53.0

### PR DESCRIPTION
**Pull request by azclibot**
Updated Azure CLI version.

### CODE REVIEW

**Comment by yonzhan** [comment]
Release

File: src/azure-cli/HISTORY.rst [hunk 1]
```diff
@@ -3,6 +3,100 @@
...
+* `az deployment group create`: Make `--template-file` parameter optional when used with `.bicepparam` parameter file (#27311)
+
+**Backup**
+
+* `az backup backup-properties`: Add option for setting `--soft-delete-feature-state` to "AlwaysOn", and `--soft-delete-duration` with values between 14 to 180 (inclusive) (#27329)
+* `az backup vault list-soft-deleted-containers`: List all soft-deleted containers in a backup vault (#27329)
+
+**Compute**
+
+* `az vm extension set`, `az vmss extension set`: Enable Autoupgrade by default for CodeIntegrityAgent extension (#27335)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:38]:
```suggestion
* `az vm/vmss extension set`: Enable Autoupgrade by default for CodeIntegrityAgent extension (#27335)
```

File: src/azure-cli/HISTORY.rst [hunk 2]
```diff
@@ -3,6 +3,100 @@
...
+* `az sql failover-group set-primary`: Add parameter `--try-planned-before-forced-failover` to support hybrid geo-failover (#27298)
+
+**Storage**
+
+* Fix #26732: `az storage blob copy start-batch`: Add `--rehydrate-priority` to batch copy (#27325)
+* Fix #27052: `az storage blob delete-batch`: Use utc as default timezone to remove "Datetime with no tzinfo will be considered UTC." warning (#27366)
+
+**Storage auto-grow**
+
+* `az postgres flexible-server create/update`: Add capability to enable/disable storage auto-grow during creation and update (#27417)
```
Comment (jiasli) [inline src/azure-cli/HISTORY.rst:95]: https://github.com/Azure/azure-cli/pull/27417's History Notes is corrupted.

File: src/azure-cli/HISTORY.rst [hunk 3]
```diff
@@ -3,6 +3,94 @@
...
+
+**Cosmos DB**
+
+* `az cosmosdb postgres`: GA Cosmos DB for PostgreSQL (#27399)
+
+**Monitor**
+
+* `az monitor action-group`: Migrate from codegen V2 (#25324)
+* `az monitor autoscale`: Migrate from codegen V2 (#27171)
+* `az monitor autoscale update`: Migrate from codegen V2 (#27338)
```
Comment (wangzelin007) [inline src/azure-cli/HISTORY.rst:57]:
```suggestion
```

File: src/azure-cli/HISTORY.rst [hunk 4]
```diff
@@ -3,6 +3,88 @@
...
+* `az cosmosdb postgres`: GA Cosmos DB for PostgreSQL (#27399)
+
+**MySQL**
+
+* `az mysql flexible-server replica create`: Add new parameters to support replica creation (#27386)
+
+**NetAppFiles**
+
+* `az netappfiles volume`: Add new command `get-groupid-list-for-ldapuser` to Get Group Id List for LDAP User (#27316)
+* `az netappfiles account update`: Add paramter `identity_type` (#27316)
```
Comment (wangzelin007) [inline src/azure-cli/HISTORY.rst:60]:
```suggestion
* `az netappfiles account update`: Add parameter `--identity-type` (#27316)
```

File: src/azure-cli/HISTORY.rst [hunk 5]
```diff
@@ -3,6 +3,88 @@
...
+* `az netappfiles volume update`: Add paramter `snapshot-dir-visible`. If enabled (true) the volume will contain a read-only snapshot directory which provides access to each of the volume's snapshots (defaults to true) (#27316)
```
Comment (wangzelin007) [inline src/azure-cli/HISTORY.rst:61]:
```suggestion
* `az netappfiles volume update`: Add parameter `--snapshot-dir-visible`. If enabled (true) the volume will contain a read-only snapshot directory which provides access to each of the volume's snapshots (defaults to true) (#27316)
```

**Review by jiasli — APPROVED** [review]

**Review by wangzelin007 — APPROVED** [review]

**Review by evelyn-ys — APPROVED** [review]

**Review by kairu-ms — APPROVED** [review]

**Review by bebound — APPROVED** [review]

**Review by AllyW — APPROVED** [review]

**Pull request merged**